webhook/index.ts

```diff
@@ -0,0 +1,117 @@
+import { createWriteStream } from 'fs'
+
+import { v4 as uuidv4 } from 'uuid'
+import express from 'express'
+import execa from 'execa'
+import Bugsnag from '@bugsnag/js'
+import BugsnagPluginExpress from '@bugsnag/plugin-express'
+
+import { WebhookConfig, exists } from './utils'
+
+const { mkdir, readdir, readFile, writeJSON } = require('fs-promise')
+
+const {
+  BUGSNAG_APP_TYPE,
+  BUGSNAG_RELEASE_STAGE,
+  EXPRESS_BUGSNAG_NOTIFIER_KEY,
+  TEMP_DIR
+} = require('../consts.json')
+
+// fs-promise only exports cjs
+
+// Bugsnag will automatically report errors from Express via a wrapper.
+Bugsnag.start({
+  apiKey: EXPRESS_BUGSNAG_NOTIFIER_KEY,
+  appType: BUGSNAG_APP_TYPE,
+  appVersion: require('../package.json').version,
+  plugins: [BugsnagPluginExpress],
+  releaseStage: BUGSNAG_RELEASE_STAGE
+})
+
+// Check version before continuing
+if (parseInt(process.versions.node) < 16) {
+  throw Error('Node version must be 16 or greater')
+}
+
+const app: express.Application = express()
+const bugsnagMiddleware = Bugsnag.getPlugin('express')
+
+const PORT = 3000
+
+// Add bugsnag middleware to automatically catch errors
+app.use(bugsnagMiddleware.requestHandler)
+app.use(express.json())
+app.use(bugsnagMiddleware.errorHandler)
+
+/**
+ * POST HTTP Endpoint. Requires specific objects in the request body, delivered by Datatools server
+ * job designed to work with this endpoint.
```

Comment:
Ref to future location of file in dev branch might be nice: https://github.com/ibi-group/datatools-server/blob/dev/src/main/java/com/conveyal/datatools/manager/jobs/PeliasUpdateJob.java (note: this is currently a broken link, but will work once https://github.com/ibi-group/datatools-server/pull/408 is merged.

Reply:
Good call! Added in bd0dd663a5578f8c305459c775f3a7881844d257